tutorials/beginners-cookbook/index.md

```diff
@@ -293,10 +293,11 @@ var interGeo = geometry1.intersection(geometry2);
 var unGeo = geometry1.union(geometry2);
 ```
 
-#### Example: Geometry operations
+#### Example: Geometry operations [[Open In Code Editor](https://code.earthengine.google.com/7f6026c7a838456212287e7e19589fcf)]
```

Comment:
Please remove Code Editor links.

Reply:
"Please don't exclude Code Editor links."

Should I keep them or remove them?

Reply:
@gino-m

Reply:
Removed the link to the code editor.